br/pkg/task/restore.go

```diff
@@ -210,6 +210,12 @@ func (cfg *RestoreConfig) ParseStreamRestoreFlags(flags *pflag.FlagSet) error {
 	if cfg.FullBackupStorage, err = flags.GetString(FlagStreamFullBackupStorage); err != nil {
 		return errors.Trace(err)
 	}
+
+	if cfg.StartTS > 0 && len(cfg.FullBackupStorage) > 0 {
+		return errors.Annotatef(berrors.ErrInvalidArgument, "just need fill one for %v and %v.",
```

Comment:
Generally, we'd better not to end the error information with dot. (Because we may annotate the error again, if there is a dot, we may get message like `something went wrong.: another thing went wrong.: ErrorCode`)
```suggestion
		return errors.Annotatef(berrors.ErrInvalidArgument, "%v and %v are mutually exclusive",
```